<!DOCTYPE html>
<html lang="en">
    <head>
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Research findings: going deeper than the article</title>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <meta name="author" content="Martin Fenner" />

    <!-- RDFa Metadata (in DublinCore) -->
    <meta property="dc:title" content="Research findings: going deeper than the article" />
    <meta property="dc:creator" content="" />
    <meta name="citation_publication_date" content="2013-12-05T00:00:00+00:00"/>
    <meta property="dc:format" content="text/html" />
    <meta property="dc:language" content="en" />
    <meta property="dc:identifier" content="/blog/2013/12/05/research-findings-going-deeper-than-the-article/" />
    <meta property="dc:rights" content="CC-BY" />
    <meta property="dc:source" content="Article-Level Metrics" />
    <meta property="dc:subject" content="Scholarly Communication" />
    <meta property="dc:type" content="website" />

    <!-- Google Scholar Metadata -->
    <meta name="citation_author" content="Jennifer Lin"/>
    <meta name="citation_author" content="Martin Fenner"/>
    <meta name="citation_publication_date" content="2013-12-05"/>
    <meta name="citation_title" content="Research findings: going deeper than the article"/>
    <meta name="citation_journal_title" content="Article-Level Metrics"/>

    <link rel="alternate" type="application/rss+xml" title="Article-Level Metrics" href="/rss.xml" />
    <link rel="alternate" type="application/atom+xml" title="Article-Level Metrics" href="/atom.xml" />

    <!-- Enable responsive viewport -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Fonts -->
    <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,400italic,600italic' rel='stylesheet' type='text/css'>

    <!-- CSS -->
    <link rel="stylesheet" href="/css/bootstrap.min.css">
    <link rel="stylesheet" href="/css/bootstrap-theme.css">
    <link rel="stylesheet" href="/css/style.css">
    <link rel="stylesheet" href="/css/github.css">

    <!-- Javascript -->
    <script src="/js/d3.v3.min.js"></script>
  </head>
  <body>
    <nav class="navbar navbar-inverse navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="#">ALM</a>
    </div>
    <div class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown">About <span class="caret"></span></a>
          <ul class="dropdown-menu">
            <li><a href="/">Introduction</a></li>
            <li><a href="/metrics">Metrics</a></li>
          </ul>
        </li>
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown">Implementations <span class="caret"></span></a>
          <ul class="dropdown-menu">
            <li><a href="/plos">Public Library of Science</a></li>
          </ul>
        </li>
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown">Development <span class="caret"></span></a>
          <ul class="dropdown-menu">
            <li><a href="/docs/">Home</a></li>
            <li><a href="/docs/installation">Installation</a></li>
            <li><a href="/docs/setup">Setup</a></li>
            <li><a href="/docs/sources">Sources</a></li>
            <li><a href="/docs/api">API</a></li>
            <li><a href="/docs/rake">Rake</a></li>
            <li><a href="/docs/alerts">Alerts</a></li>
            <li><a href="/docs/faq">FAQ</a></li>
            <li><a href="/docs/releases">Releases</a></li>
            <li><a href="/docs/roadmap">Roadmap</a></li>
            <li><a href="/docs/contributors">Contributors</a></li>
          </ul>
        </li>
        <li><a href="/examples">Examples</a></li>
        <li><a href="/blog">Blog</a></li>
        <li><a href="/status">Status</a></li>
      </ul>
    </div>
  </div>
</nav>
    <div class="container">
      <div class="page-header">
        <h1>Research findings: going deeper than the article</h1>
        <p class="subtitle">05 December 2013 by
          
<a href="mailto:jlin@plos.org">Jennifer Lin</a> and 
<a href="http://orcid.org/0000-0003-1419-2405">Martin Fenner</a>
        </p>
      </div>

      <div class="content">
        <div class="row">
          <div class="col-md-9">
            
            <p>We have recently released two new Article-Level Metrics (ALM) data
sources: <strong>Europe PMC Database Citations</strong> and <strong>DataCite</strong>. The data
from both sources are displayed on the metrics tab of PLOS articles, are
available through the <a href="http://api.plos.org/alm/using-the-alm-api/">PLOS ALM
API</a>, and are available to
all users of the <a href="https://github.com/articlemetrics/alm">open source ALM
application</a>. These new sources
are both related to research data, and they represent a new breed of
metrics in our ALM suite.</p>

<p>Not only are <strong>Europe PMC Database Citations</strong> and **DataCite** our
first ALM sources that track research data that mention a particular
PLOS article. But these ALM sources are also different from other ALM
sources: although there can of course be additional research data that
cite an article post-publication*,* they typically link datasets
associated with an article and created by the same research
group.* *These ALM sources discover links <em>from the research data to the
article</em> and in an ideal world should be consistent with the links <em>from
the article to the research data</em>. Unfortunately we know from the work
by Jo McEntyre and others at Europe PMC (see the May 2013 <a href="http://dx.doi.org/10.1371/journal.pone.0063184">PLOS ONE
article by Kafkas et
al</a>.) that the overlap
between <em>article-to-database</em> citations and <em>database-to-article</em>
citations is surprisingly small.</p>

<p>PLOS and other publishers should of course do a better job helping
authors to properly cite research data in their submitted manuscripts,
and the recently published <a href="http://codata.org/blog/2013/11/25/data-citation-synthesis-group-draft-declaration-of-data-citation-principles/">Draft Declaration of Data Citation
Principles</a>
are an excellent starting point. But providing the links from
database-to-article as ALM can also increase the visibility of datasets
associated with an article. We know of course that we are not the first
publisher to do this, earth and environmental sciences research data
deposited in the Pangaea data archive and cited in Elsevier articles
<a href="http://wiki.pangaea.de/wiki/Elsevier">were highlighted since 2009</a>
using a similar mechanism – taking advantage of the database-to-article
link provided by the DataCite DOI service.</p>

<h2 id="toc_0">Europe PMC Database Citations</h2>

<p><a href="http://europepmc.org">Europe PubMed Central</a> is a service of the Europe
PMC Funders’ Group working in partnership with the European
Bioinformatics Institute, University of Manchester and the British
Library in cooperation with the National Center for Biotechnology
Information at the U.S. National Library of Medicine (NCBI/NLM). Europe
PMC is not only hosting many important databases in the life sciences in
collaboration with PubMed and others – e.g., UniProt, Protein Data Bank
(PDB), and the European Nucleotide Archive (ENA) – but is also
extracting the citations to journal articles in these datasets and makes
them available via a public API. We harvest these citations in aggregate
form and incorporate them into our newly expanded suite of ALMs. As with
all ALMs (to the extent allowable by source), we link out to Europe PMC
so that users can access the data from the source.</p>

<p>The <a href="http://www.uniprot.org/uniprot/I0XPC4?europe_pmc_bio_extredirect_Proteins=http://www.uniprot.org/uniprot/I0XPC4">methylthiotransferase protein
entry</a>
in the UniProtKB/TrEMBL database cites a <a href="http://www.plosntds.org/article/info%3Adoi%2F10.1371%2Fjournal.pntd.0001853">PLOS NTD
publication</a>.</p>

<p><a href="http://www.uniprot.org/uniprot/I0XPC4?europe_pmc_bio_extredirect_Proteins=http://www.uniprot.org/uniprot/I0XPC4"><img src="/assets/ExDBCite-1024x544.png" alt="Europe PMC DB citation example"></a></p>

<p>Different public databases will have varying levels of metadata
available and interconnecting links from data to published results.
EMBL’s European Nucleotide Archive is a good way forward as you can see
in this example taken from the <a href="http://www.ebi.ac.uk/ena/data/view/HE663067?europe_pmc_bio_tm_extredirect_Data_Citations=http://www.ebi.ac.uk/ena/data/view/HE663067">Mycobacterium tuberculosis
genome</a>
.</p>

<p><img src="/assets/ExamplePMCEuropeLink.png" alt="ENA Example"></p>

<h2 id="toc_1">DataCite</h2>

<p><a href="http://www.datacite.org/">DataCite</a> is a DOI registration agency with
the aim to *establish easier access to research data on the Internet*
and to *increase acceptance of research data as legitimate, citable
contributions to the scholarly record.*Although DataCite focuses on
research data, they provide DOIs for all forms of content, e.g. <a href="https://peerj.com/preprints/">PeerJ
Preprints</a> or all non-publisher content on
<a href="http://figshare.com">Figshare</a>. Most people are familiar with CrossRef
DOIs, but they are not the only DOI registration agency (see Geoff
Bilder’s September <a href="http://crosstech.crossref.org/2013/09/dois-unambiguously-and-persistently-identify-published-trustworthy-citable-online-scholarly-literature-right.html">blog
post</a>
for more background info). CrossRef and DataCite do of course
collaborate, e.g. in <a href="http://crosscite.org/cn/">DOI content
negotiation</a>.</p>

<p>We have provided CrossRef ALM for a very long time, and have now added
DataCite as ALM source. The <a href="http://schema.datacite.org/">DataCite metadata
schema</a> is different from CrossRef’s: we
are searching the <em>relatedIdentifier</em> metadata for PLOS DOIs and as of
today we find 240 DataCite DOIs related to PLOS papers. Similar to the
Europe PMC Database Links, most of these datasets are associated with a
paper and submitted by the same research group. Most DataCite DOIs
linking to PLOS papers are from datasets deposited into the
<a href="http://datadryad.org/">Dryad</a> data repository (disclaimer: Martin is a
member of the Dryad Board), followed by
<a href="http://www.pangaea.de/">Pangaea</a>.</p>

<p>The 2011 paper by Jonathan Eisen et al., <a href="http://dx.doi.org/10.1371/journal.pone.0018011">Stalking the Fourth Domain in
Metagenomic Data</a>, has
underlying data deposited in Dryad, which is a member repository and is
thus <a href="http://search.datacite.org/ui?q=relatedIdentifier:10.1371%2Fjournal.pone.0018011">linked in
DataCite</a>.
Alternatively, we can see that <a href="http://data.datacite.org/10.1594/PANGAEA.802516">the “Biogeochemical measurements
associated to deep-sea wood
falls” dataset</a> is a
supplement to the <a href="http://dx.doi.org/10.1371/journal.pone.0053590">2013 Antje Boetius, et. al
paper</a> and is cited as
supporting data which informed the research conclusions published. Both
PLOS papers also link to the dataset(s) using the DataCite DOI in the
M*aterials and Methods* section, as recommended by the <a href="http://www.plosone.org/static/editorial#sharing">PLOS Editorial
Policies</a>.</p>

<h2 id="toc_2">Linking Research Data and Journal Articles (and People)</h2>

<p>We made a conscious decision to display both of these data-related ALMs
in the Citations category. The <a href="http://www.force11.org/datacitation">Draft Declaration of Data Citation
Principles</a> states that*Data
citations should be accorded the same importance in the scholarly record
as citations of other research objects, such as publications.*We think
that the inverse is also true, i.e. citations of publications by
datasets should have the same importance in the scholarly record and
should be grouped together with citations by other publications. Data
citation in both directions can enable easy reuse and verification of
data, allow the impact of data to be tracked, and create a scholarly
structure that recognizes and rewards data producers.</p>

<p>As a publisher, we call for a dedicated space on the article to
seamlessly link to the public repositories where the data are available.
At PLOS, there is much work to do as this <a href="http://www.plosone.org/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0041389">example from Jonathan Eisen,
et
al</a>clearly
demonstrates. Eisen has diligently <a href="http://phylogenomics.wordpress.com/data/phylogenomics-of-halophiles-data-from-lynch-et-al-2012/">catalogued all the datasets from
this paper on his own
blog.</a>
But few researchers have employed this practice. More importantly, we
need to display and preserve this information as part of the published
record.</p>

<p>The careful observer will have noted that the two example articles by
Jonathan Eisen and Antje Boetius mentioned above also have citations by
other
publications (<a href="http://europepmc.org/abstract/MED/21437252#fragment-related-citations">here</a> and <a href="http://europepmc.org/abstract/MED/23301092#fragment-related-citations">here</a>) that
are tracked in <strong>Europe PMC Citations</strong>, another new ALM source. And
that Europe PMC links to their ORCID author identifier
(<a href="http://orcid.org/0000-0002-0159-2197">here</a> and
<a href="http://orcid.org/0000-0003-2117-4176">here</a>). To go full circle they
can use the <a href="http://datacite.labs.orcid-eu.org/">DataCite/ORCID integration
tool</a> to also import their Dryad or
Pangaea datasets into their ORCID profile, and then can use <a href="http://feed.labs.orcid-eu.org">ORCID
content negotiation</a> (disclaimer: Martin
was involved in the development of both ORCID tools) to automatically
generate a <a href="http://feed.labs.orcid-eu.org/0000-0002-0159-2197.rss">RSS
feed</a> or BibTeX
reference manager file of all their publications and datasets.</p>

<p>In this light, we anticipate that the upcoming Data Literature
Integration Workshop at EMBL-EBI headed by Jo McEntyre, Thomas
Lemberger, and Ewan Birney on December 10-11, 2013 will address many of
the much-needed issues applicable across our community and lay down
better practices for handling data-publication interactions.</p>



            <!-- COinS metadata (for citation managers like Zotero etc), goes in body text -->
<span
  class="Z3988"
  title="ctx_ver=Z39.88-2004
  &amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Adc
  &amp;rfr_id=info%3Asid%2Focoins.info%3Agenerator
  &amp;rft.title=Research findings: going deeper than the article
  &amp;rft.creator=
  &amp;rft.date=2013-12-05
  &amp;rft.language=EN
  &amp;rft.rights=CC0
  &amp;rft_id=http://articlemetrics.github.io/blog/2013/12/05/research-findings-going-deeper-than-the-article/">
</span>

            
              <hr>
              <div id="disqus_thread"></div>
<script type="text/javascript">
    
    var disqus_shortname = 'articlemetrics'; // required: replace example with your forum shortname
    
    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="http://disqus.com" class="dsq-brlink">blog comments powered by <span class="logo-disqus">Disqus</span></a>
            
            <hr>
            <footer class="footer" role="contentinfo">
  <iframe src="http://ghbtns.com/github-btn.html?user=articlemetrics&repo=alm&type=watch&count=true"
  allowtransparency="true" frameborder="0" scrolling="0" width="100" height="20"></iframe>

  <iframe class="github-btn" src="http://ghbtns.com/github-btn.html?user=articlemetrics&repo=alm&type=fork&count=true"
  allowtransparency="true" frameborder="0" scrolling="0" width="100" height="20"></iframe>

  <iframe src="http://ghbtns.com/github-btn.html?user=articlemetrics&type=follow&count=true"
  allowtransparency="true" frameborder="0" scrolling="0" width="180" height="20"></iframe>

  <p>Maintained by the core team with the help of our <a href="/docs/Contributors">contributors</a>.<br/>
  Source code for <a href="https://github.com/articlemetrics/alm">ALM application</a> and <a href="https://github.com/articlemetrics/articlemetrics.github.io">this website</a> available on Github. <br/> Code licensed under <a href="https://github.com/articlemetrics/alm/blob/master/LICENSE.md">Apache 2.0</a>, documentation under <a href="http://creativecommons.org/licenses/by/4.0/">CC-BY 4.0</a>.</p>
</div>
          </div>
        </div>


      </div>
    </div> <!-- /container -->

    <script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', '']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>


    <script src="http://code.jquery.com/jquery.js"></script>
    <script src="/js/bootstrap.min.js"></script>
    <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  </body>
</html>